apps/src/music/views/MusicView.jsx

```diff
@@ -671,11 +672,16 @@ class UnconnectedMusicView extends React.Component {
               this.renderInstructions(InstructionsPositions.LEFT)}
 
             <div id="blockly-area" className={moduleStyles.blocklyArea}>
+              {(this.props.inIncubator ||
+                this.props.progressLevelType === ProgressLevelType.LEVEL) && (
```

Comment:
Maybe we can hoist this to a helper function, like `hasNoProgressHeader` (or simpler), akin to the existing `hasProgression` helper?